### PR TITLE
fix(ENG-830): Fix average # of files changed in reviews sections

### PR DESCRIPTION
**Pull request by navassouza**
Closes: https://athenianco.atlassian.net/browse/ENG-830

### CODE REVIEW

File: src/js/components/insights/stages/review/pullRequestSize.jsx [hunk 1]
```diff
@@ -56,15 +56,18 @@ const pullRequestSize = {
                     };
                 })
                 .orderBy(['age', 'loc'], ['desc', 'desc'])
-                .take(20)
+                // .take(20)
```
Comment (se7entyse7en) [inline src/js/components/insights/stages/review/pullRequestSize.jsx:59]: Leftover?

Comment (navassouza) [reply src/js/components/insights/stages/review/pullRequestSize.jsx:59]: Fixed. I was trying to understand why we'd want only the first 20 entries.

Comment (warenlg) [reply src/js/components/insights/stages/review/pullRequestSize.jsx]: Wait, but we can not display all the PRs in the chart, sometimes with have hundreds, that's why we took the top 20.

Comment (navassouza) [reply src/js/components/insights/stages/review/pullRequestSize.jsx]: Added back, this piece of code has some processing waste, I'll address it later

**Comment by warenlg** [comment]
I don't understand, is this PR fixing https://athenianco.atlassian.net/browse/ENG-830?

**Comment by navassouza** [comment]
@warenlg yes, lines 62, 64 address the #, I wasn't sure about the limit and commented it out, should have asked. But you have it back

**Comment by warenlg** [comment]
Ok my bad, I think I got it!
I was confused because you edited the 1st KPI box and not the 2nd that is the topic of the ticket. But now I got it, you changed the way `totalFiles` is computer.

**Pull request merged**